EXILED/Exiled.API/Features/Map.cs

```diff
@@ -130,7 +125,7 @@ public static Scp939AmnesticCloudInstance AmnesticCloudPrefab
         /// <summary>
         /// Gets all <see cref="AdminToy"/> objects.
         /// </summary>
-        public static ReadOnlyCollection<AdminToy> Toys { get; } = ToysValue.AsReadOnly();
+        public static ReadOnlyCollection<AdminToy> Toys => AdminToy.BaseToAdminToy.Values.ToList().AsReadOnly();
```

Comment:
are you sure to make that list so inefficiently?

`Dictianary.ValueCollection` is inherited from `ReadOnlyCollection`, so we can just

```suggestion
        public static ReadOnlyCollection<AdminToy> Toys => AdminToy.BaseToAdminToy.Values;
```

Reply:
Oh i missed that it is `ReadOnlyCollection`, not `IReadOnlyCollection` interface, we cant do it more memory efficient😭

Reply:
yup anyway this will be deleted for Exiled9